src/index.html

```diff
@@ -3,82 +3,90 @@
+      <ul class="products" role="list">
+        <li class="card">
           <img
+            alt="White robot"
             src="https://images.unsplash.com/photo-1485827404703-89b55fcc595e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=200&h=200&q=60"
           />
-          <h2>Item 1</h2>
+          <h3 class="h2">Item 1</h3>
           <p>
             Sesame snaps gummi bears macaroon sweet chocolate cake lemon drops
             donut. Marzipan lollipop gummies chocolate bar jelly gummies icing
             oat cake.
           </p>
-          <button class="button">Buy Now</button>
-        </div>
-        <div class="card">
+          <a href="#" class="button" aria-label="Buy Item 1 Now">Buy Now</a>
+        </li>
+        <li class="card">
           <img
+            alt="Drone hovering slightly above a human hand"
             src="https://images.unsplash.com/photo-1504890001746-a9a68eda46e2?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&h=200&q=60"
           />
-          <h2>Item 2</h2>
+          <h3 class="h2">Item 2</h3>
           <p>
             Biscuit chocolate powder cheesecake powder candy canes soufflé
             lollipop. Macaroon donut danish candy canes icing.
           </p>
-          <button class="button">Buy Now</button>
-        </div>
-        <div class="card">
+          <a href="#" class="button" aria-label="Buy Item 2 Now">Buy Now</a>
+        </li>
+        <li class="card">
           <img
+            alt="Laptop on a desk"
             src="https://images.unsplash.com/photo-1496181133206-80ce9b88a853?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=200&h=200&q=60"
           />
-          <h2>Item 3</h2>
+          <h3 class="h2">Item 3</h3>
           <p>
             Caramels chocolate halvah ice cream macaroon cupcake. Chupa chups
             cotton candy lemon drops. Carrot cake brownie topping biscuit.
           </p>
-          <button class="button">Buy Now</button>
-        </div>
-      </aside>
+          <a href="#" class="button" aria-label="Buy Item 3 Now">Buy Now</a>
+        </li>
+      </ul>
 
-      <div class="form">
+      <form class="form">
         <h2>Sign Up</h2>
 
-        <label>Name</label>
-        <input type="text" />
+        <label for="name">Name</label>
+        <input type="text" id="name" name="name" />
 
-        <label>Email</label>
-        <input type="email" />
+        <label for="email">Email</label>
+        <input type="email" id="email" name="email" />
 
+        <label for="message">Message</label>
         <textarea
-          name=""
-          id=""
+          name="message"
+          id="message"
           cols="30"
           rows="3"
-          placeholder="Message"
         ></textarea>
 
-        <button class="button" role="button">Submit</button>
-      </div>
-    </div>
-    <div class="footer">
+        <button class="button" role="submit">Submit</button>
```

Comment:
Small error here!

```suggestion
        <button class="button" type="submit">Submit</button>
```

Reply:
Good catch! I saw this in your suggested changes and it makes sense. Thank you! Fixed with `commit b6d61a5`